beacon_node/http_api/src/lib.rs

```diff
@@ -1709,6 +1709,41 @@ pub fn serve<T: BeaconChainTypes>(
         .and(warp::path("rewards"))
         .and(chain_filter.clone());
 
+    // POST beacon/rewards/attestations/{epoch}
+    let post_beacon_rewards_attestations = beacon_rewards_path
+        .clone()
+        .and(warp::path("attestations"))
+        .and(warp::path::param::<Epoch>())
+        .and(warp::path::end())
+        .and(warp::body::json())
+        .and(log_filter.clone())
+        .and_then(
+            |chain: Arc<BeaconChain<T>>, epoch: Epoch, validators: Vec<usize>, log: Logger| {
```

Comment:
We should accept a `Vec<ValidatorId>` which will allow unquoted indices, quoted indices, and public keys.

```suggestion
            |chain: Arc<BeaconChain<T>>, epoch: Epoch, validators: Vec<ValidatorId>, log: Logger| {
```

We can convert the IDs to validator indices before calling the underlying function using logic like this:

https://github.com/sigp/lighthouse/blob/0866b739d0cb4b7974e70cbd5b67388f36cd1361/beacon_node/http_api/src/sync_committee_rewards.rs#L35-L41